docs/usage/flower17/flower17_scikit.rst

```diff
@@ -0,0 +1,108 @@
++---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|  **Haralick**       |  This operation extracts the texture features from the image.                                                                                                                             |
++---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|  **flatten**        |  This operation returns a copy of numpy array collapsed into 1 dimension.                                                                                                                 |
++---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+The dataflow visualised:
+
+.. image:: /../examples/flower17/sklearn-opencv/dataflow_diagram.svg
+
+We then create the dataflow config file, using the ``dataflow create`` command.
+
+.. literalinclude:: /../examples/flower17/sklearn-opencv/create_dataflow.sh
+
+To re-create the visualization of the dataflow above, run:
+
+.. code-block:: console
+
+    dffml dataflow diagram features.yaml -simple -stages processing -configloader yaml
+
+Copy and pasting the output of the above code into the
+`mermaidjs live editor <https://mermaidjs.github.io/mermaid-live-editor>`_
+results in the graph.
+
+Train the model.
+
+.. literalinclude:: /../examples/flower17/sklearn-opencv/train.sh
+
+Assess the model's accuracy.
+
+.. literalinclude:: /../examples/flower17/sklearn-opencv/accuracy.sh
+
+The output is:
+
+.. code-block:: console
+
+    0.27450980392156865
+
+Create an ``unknown_images.csv`` file which contains the filenames of the images to predict on.
+
+.. literalinclude:: /../examples/flower17/unknown_data.sh
+
+In this example, the ``unknown_images.csv`` file contains the filenames of the following images
+
+.. image:: /../examples/flower17/daisy.jpg
+    :width: 140px
+    :height: 140px
+
+.. image:: /../examples/flower17/pansy.jpg
+    :width: 140px
+    :height: 140px
+
+.. image:: /../examples/flower17/tigerlily.jpg
+    :width: 140px
+    :height: 140px
+
+.. image:: /../examples/flower17/buttercup.jpg
+    :width: 140px
+    :height: 140px
+
+Predict with the trained model.
+
+.. literalinclude:: /../examples/flower17/sklearn-opencv/predict.sh
+
+Output
+
+.. literalinclude:: /../examples/flower17/sklearn-opencv/output.txt
+
+The model predicts 2 of the 4 images correctly!
+
+The accuracy of the model can be increased by using various preprocessing techniques
+for example scaling all the feature vectors within the range [0,1].
```

Comment:
We should end by linking to the CNN tutorial and saying that that will vastly improve accuracy

Reply:
Sounds good!